code/ai/aigoals.cpp

```diff
@@ -1983,7 +1983,7 @@ ai_achievability ai_mission_goal_achievable( int objnum, ai_goal *aigp )
 			else if ( status == SHIP_STATUS_UNKNOWN )
 				return ai_achievability::NOT_KNOWN;
 
-			Int3();		// get allender -- bad logic
+			UNREACHABLE("get allender -- bad logic");		// get allender -- bad logic
```

Comment:
We could dump the status variable here.  "Get allender -- I mean, get the SCP" would direct people better, too.

Reply:
Good idea